src/libs/GooglePlacesUtils.js

```diff
@@ -21,41 +21,7 @@ function getAddressComponent(addressComponents, type, key) {
         .value();
 }
 
-/**
- * Validates this contains the minimum address components
- *
- * @param {Array} addressComponents
- * @returns {Boolean}
- */
-function isAddressValidForVBA(addressComponents) {
```

Comment:
we removed this in favor of our address check in `ValidationUtils`?

Reply:
The address check in `ValidationUtils` has always been there I think. Before, we were using `isAddressValidForVBA` to make sure that the result the user clicked had all the information we needed. If it was missing something, we rejected the result. The result was rejected because other address fields were hidden from the user and the user was not able to complete what was missing.

The approach changed here a bit, since the user can always edit all parts of the address, we don't care anymore about the address result being complete or not. We will just use the information available and ignore the missing parts. The user can fill the missing parts later and if he doesn't he will get validation errors on submit.

Reply:
Got it, thanks for the clarification!